src/main/java/domain/Ladder.java

```diff
@@ -0,0 +1,62 @@
+package domain;
+
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Deque;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class Ladder {
+    private static final String ERROR_SAME_LINE_SIZE = "[ERROR] 다른 길이의 가로줄이 생성되었습니다";
+    private static final String ERROR_LINE_SIZE_EMPTY = "[ERROR] 길이가 0인 가로줄이 생성되었습니다";
+
+    private final List<Line> lines;
+
+    public Ladder(final Width width, final Height height, final ScaffoldGenerator scaffoldGenerator) {
+        this.lines = IntStream.range(0, height.getValue())
+                .mapToObj(it -> createLine(width, scaffoldGenerator))
```

Comment:
바로 new Line 을 호출하지 않은 이유가 있었을까요 🤔 

```suggestion
                .mapToObj(it -> new Line(width, scaffoldGenerator)
```